File: SEMANA5_PY/ejer4.py
cant=int(input("Ingrese la cantidad de números: "))
pares=impares=ceros=0

print()
for i in range(1, cant+1):
    num=int(input(f"Ingrese número {i}: "))

    if num==0:
        ceros+=1 # ceros++
    elif num%2==0:
        pares+=1
    else:
        impares+=1

print("# pares: ",pares)
print("# impares: ",impares)
print("# ceros: ",ceros)

File: SEMANA11_YORDAN/ejer4.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SEMANA11_YORDAN
{
    internal class ejer4
    {
        static void Main(string[] args)
        {
            int[] num = new int[10];
            for (int i = 0; i < num.Length; i++)
            {
                Console.Write($"Ingrese número {i + 1}: ");
                num[i] = int.Parse(Console.ReadLine());

            }
            Console.Write("\nNúmeros ingresados: ");
            foreach (int i in num)
            {
                Console.Write(i+" ");
            }
            Console.WriteLine("\nNúmeros pares: ");
            for(int i=0; i< num.Length; i++)
            {
                if (num[i] % 2 == 0)
                {
                    Console.Write(num[i] + " ");
                }
            }
            Console.ReadKey();
        }
    }
}
